src/devices/Amg88xx/Register.cs

```diff
@@ -0,0 +1,951 @@
+        /// Pixel 53 value register (low byte)
+        /// </summary>
+        T53L = 0xe8,
+
+        /// <summary>
+        /// Pixel 53 value register (high byte)
+        /// </summary>
+        T53H = 0xe9,
+
+        /// <summary>
+        /// Pixel 54 value register (low byte)
+        /// </summary>
+        T54L = 0xea,
+
+        /// <summary>
+        /// Pixel 54 value register (high byte)
+        /// </summary>
+        T54H = 0xeb,
+
+        /// <summary>
+        /// Pixel 55 value register (low byte)
+        /// </summary>
+        T55L = 0xec,
+
+        /// <summary>
+        /// Pixel 55 value register (high byte)
+        /// </summary>
+        T55H = 0xed,
+
+        /// <summary>
+        /// Pixel 56 value register (low byte)
+        /// </summary>
+        T56L = 0xee,
+
+        /// <summary>
+        /// Pixel 56 value register (high byte)
+        /// </summary>
+        T56H = 0xef,
+
+        /// <summary>
+        /// Pixel 57 value register (low byte)
+        /// </summary>
+        T57L = 0xf0,
+
+        /// <summary>
+        /// Pixel 57 value register (high byte)
+        /// </summary>
+        T57H = 0xf1,
+
+        /// <summary>
+        /// Pixel 58 value register (low byte)
+        /// </summary>
+        T58L = 0xf2,
+
+        /// <summary>
+        /// Pixel 58 value register (high byte)
+        /// </summary>
+        T58H = 0xf3,
+
+        /// <summary>
+        /// Pixel 59 value register (low byte)
+        /// </summary>
+        T59L = 0xf4,
+
+        /// <summary>
+        /// Pixel 59 value register (high byte)
+        /// </summary>
+        T59H = 0xf5,
+
+        /// <summary>
+        /// Pixel 60 value register (low byte)
+        /// </summary>
+        T60L = 0xf6,
+
+        /// <summary>
+        /// Pixel 60 value register (high byte)
+        /// </summary>
+        T60H = 0xf7,
+
+        /// <summary>
+        /// Pixel 61 value register (low byte)
+        /// </summary>
+        T61L = 0xf8,
+
+        /// <summary>
+        /// Pixel 61 value register (high byte)
+        /// </summary>
+        T61H = 0xf9,
+
+        /// <summary>
+        /// Pixel 62 value register (low byte)
+        /// </summary>
+        T62L = 0xfa,
+
+        /// <summary>
+        /// Pixel 62 value register (high byte)
+        /// </summary>
+        T62H = 0xfb,
+
+        /// <summary>
+        /// Pixel 63 value register (low byte)
+        /// </summary>
+        T63L = 0xfc,
+
+        /// <summary>
+        /// Pixel 63 value register (high byte)
+        /// </summary>
+        T63H = 0xfd,
+
+        /// <summary>
+        /// Pixel 64 value register (low byte)
+        /// </summary>
+        T64L = 0xfe,
+
+        /// <summary>
+        /// Pixel 64 value register (high byte)
+        /// </summary>
+        T64H = 0xff
+    }
+
+    /// <summary>
+    /// Defines the status bits of the status register (addr: 0x04)
+    /// </summary>
+    public enum StatusFlagBit : byte
```

Comment:
One file per enum for readability and maintenance. 
And same question as for the previous enum, does it need to be public?